Python/compile.c

```diff
@@ -2027,26 +2027,39 @@ compiler_visit_annexpr(struct compiler *c, expr_ty annotation)
 
 static int
 compiler_visit_argannotation(struct compiler *c, identifier id,
-    expr_ty annotation, PyObject *names)
+    expr_ty annotation, PyObject *annotations)
 {
     if (annotation) {
-        PyObject *mangled;
-        VISIT(c, annexpr, annotation);
```

Comment:
Remove compiler_visit_annexpr.

Reply:
This function is used for class/module annotations.

Reply:
This is a line where it used
https://github.com/python/cpython/blob/ce04e7105bc396c32667a22b928a712ba0778a3f/Python/compile.c#L5271

Please correct me if I miss smth.

Reply:
You are right. I grepped `compiler_visit_annexpr` by mistake. Please forget it.